Makefile

```diff
@@ -222,6 +227,7 @@ travis-install-acceptance: install-yarn travis-install-postgres
 	mkdir -p ~/.bin
 	mv ~/chromedriver ~/.bin/
 travis-install-network: travis-install-postgres
+travis-install-snuba: travis-install-postgres
```

Comment:
I can't rememner why the snuba service relies on postgres?

Reply:
Snuba doesn't, but I assumed our real tests will hit the Django ORM (the only test right now is kind of a `"hello, world`").